Neo4j.Driver/Neo4j.Driver/Preview/Mapping/DoNotMapAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [http://neo4j.com]
+// 
+// This file is part of Neo4j.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Neo4j.Driver.Preview.Mapping;
+
+/// <summary>
+/// Instructs the default object mapper not to attempt to map any value to this property.
+/// This attribute does not affect custom-defined mappers.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property)]
+public class DoNotMapAttribute : Attribute
```

Comment:
Can we rename this to `MappingIgnored` or something similar, I think `DoNotMap` is quite imperative and attributes read better as declarative.